test/TestProjects/ResourceClients-LowLevel/Generated/ResourceServiceClient.cs

```diff
@@ -0,0 +1,242 @@
+#pragma warning disable AZC0002
+        public virtual Response GetGroups(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetGroups");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetGroupsRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get all groups. It is defined in `Item` subclient, but must be promoted to the `Service` client, because it has neither `groupId` nor `itemId` parameters. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual async Task<Response> GetAllItemsAsync(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetAllItems");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetAllItemsRequest(context);
+                return await _pipeline.ProcessMessageAsync(message, _clientDiagnostics, context).ConfigureAwait(false);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Get all groups. It is defined in `Item` subclient, but must be promoted to the `Service` client, because it has neither `groupId` nor `itemId` parameters. </summary>
+        /// <param name="context"> The request context, which can override default behaviors on the request on a per-call basis. </param>
+#pragma warning disable AZC0002
+        public virtual Response GetAllItems(RequestContext context = null)
+#pragma warning restore AZC0002
+        {
+            using var scope = _clientDiagnostics.CreateScope("ResourceServiceClient.GetAllItems");
+            scope.Start();
+            try
+            {
+                using HttpMessage message = CreateGetAllItemsRequest(context);
+                return _pipeline.ProcessMessage(message, _clientDiagnostics, context);
+            }
+            catch (Exception e)
+            {
+                scope.Failed(e);
+                throw;
+            }
+        }
+
+        /// <summary> Initializes a new instance of ResourceGroup. </summary>
+        /// <param name="groupId"> The String to use. </param>
+        /// <exception cref="ArgumentNullException"> <paramref name="groupId"/> is null. </exception>
+        public virtual ResourceGroup GetResourceGroupClient(string groupId)
```

Comment:
In the guidelines, we don't have the `Client` suffix on Resource Client factory methods.
https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-subclient-factory-methods-suffix

Reply:
Fixed